### PR TITLE
Modified the code so that thread finishes in time

### PR DESCRIPTION
**Pull request by shitongs**
-Modified the code so that thread finishes in time

### CODE REVIEW

File: app/src/main/java/com/deframe/artapp/ui/MapViewFragment.kt [hunk 1]
```diff
@@ -71,9 +71,12 @@ class MapViewFragment : Fragment(), OnMapReadyCallback {
         }
 
         //runs a thread to call the database
-        Thread(Runnable {
+        var t = Thread(Runnable {
```
Comment (ankurbag) [inline app/src/main/java/com/deframe/artapp/ui/MapViewFragment.kt:74]: What is the reason of this change? Is it related to the MapView Bug?

Comment (shitongs) [reply app/src/main/java/com/deframe/artapp/ui/MapViewFragment.kt:74]: Yes, please try it out and see if it still causes the bug. I cannot test on my side due to different thread running time. Thanks!

Comment (ankurbag) [reply app/src/main/java/com/deframe/artapp/ui/MapViewFragment.kt:74]: Sure.

**Review by ankurbag — APPROVED** [review]

**Comment by ankurbag** [comment]
You can merge it 👍
